cedars/app/api.py

```diff
@@ -0,0 +1,94 @@
+import os
+from loguru import logger
+from tenacity import retry, wait_exponential
+import requests
+from . import db
+
+def load_pines_url(project_id, superbio_api_token = None):
+    '''
+    if PINES_URL is not available in the ENV then
+    - Start a PINES SERVER
+    - With retry logic - keep making get requests
+    - Get request gives a PINES URL
+    - Call this URL for PINES predictions
+    '''
+
+    env_url = os.getenv("PINES_API_URL")
+    api_url = os.getenv("SUPERBIO_API_URL")
+    if env_url is not None:
+        # Get PINES api from .env
+        pines_api_url = env_url
+        is_url_from_api = False
+        logger.info(f"Received url : {pines_api_url} for pines from ENV variables.")
```

Comment:
in this case, can you hit the `/healthcheck` endpoint in PINES to make sure it is up and running? 

@app.get("/healthcheck")
async def healthcheck():
    return {"message": f"Welcome to the Pines NLP Model\n\n Current Model: {model_name}",
            "status": "Healthy"}